job-scraper/main.py

```diff
@@ -0,0 +1,27 @@
+from job_scraper import JobScraper
+import config
+
```

Comment:
Spójrz na mojego snippet'a, trochę to wszystko przerobiłem i trochę inaczej operuje na zwracanych danych, mam nadzieje, że mój kod trochę Ci bardziej rozjaśni jak można pisać taki kod. Starałaś się jedną klasą i w main'ie oporządzić wszystkie dane. Ja wszystko przeniosłem do klasy i cała serializacja + przeróbka danych dzieje się w klasach i ostatnie wywołanie w main'ie sprowadza się do dwóch linijek + jakieś wyświetlenie wyniku.